styles.css

```diff
@@ -425,6 +439,63 @@ textarea {
   display: none;
 }
 
+nav.menu-image {
+  display: block;
+}
+
+.menu-bar {
+  position: fixed;
+  z-index: 99;
+  height: 100vh;
+  width: 100vh;
+  left: 0;
+  top: 0;
+  background-color: #6070ff;
+  padding: 1em;
+
+
+  li {
+    a {
+      
+    }
+  }
+  /* width: 100%; */
+  /* height: 100%; */
+  /* margin: 0; */
+  /* padding: 0; */
+}
```

Comment:
it will be nice to remove all unused codes from your style files to make your code neat.